src/main/java/org/mule/tools/devkit/sonar/rule/verifier/java/ProcessorParametersQuantityVerifier.java

```diff
@@ -0,0 +1,24 @@
+package org.mule.tools.devkit.sonar.rule.verifier.java;
+
+import com.sun.source.tree.MethodTree;
+import com.sun.source.tree.VariableTree;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.mule.tools.devkit.sonar.Rule;
+
+import java.util.List;
+
+public class ProcessorParametersQuantityVerifier extends ConnectorClassVerifier {
+
+    public ProcessorParametersQuantityVerifier(Rule.@NonNull Documentation doc) {
+        super(doc);
+    }
+
+    @Override
+    protected void verifyProcessor(@NonNull MethodTree method, @NonNull final List<? extends VariableTree> parameters) {
+
+        if(parameters.size() >= 5){
```

Comment:
Is there any way to make this 5 configurable? I think 5 is too low, and no matter what number we choose, it will always defy reality.